apps/webstore/src/app/shared/components/items/checkout-item/checkout-item.component.ts

```diff
@@ -89,6 +91,17 @@ export class CheckoutItemComponent implements OnInit, AfterContentInit {
         }
         break;
       }
+      case DesignTypeEnum.petSign: {
```

Comment:
Since this is the same setting as for quotable you can just add it to the same case as quotable: 

case DesignTypeEnum.quotable:
case DesignTypeEnum.petSign: { ... }